sources/api/netdog/src/cli/generate_net_config.rs

```diff
@@ -15,6 +17,10 @@ pub(crate) fn run() -> Result<()> {
         net_config::from_path(DEFAULT_NET_CONFIG_FILE).context(error::NetConfigParseSnafu {
             path: DEFAULT_NET_CONFIG_FILE,
         })?
+    } else if Path::exists(Path::new(ALT_NET_CONFIG_FILE)) {
+        net_config::from_path(ALT_NET_CONFIG_FILE).context(error::NetConfigParseSnafu {
+            path: ALT_NET_CONFIG_FILE,
+        })?
```

Comment:
The downside of this ordering is that it becomes impossible for a bootstrap container to override a copy of `net.toml` provided at provisioning time, or through other means.

That would create a feature gap between `metal` variants, which generally couldn't override via bootstrap container, and `aws` and `vmware` variants, which could.

It might be better to reverse the order and adjust the naming:
* `OVERRIDE_NET_CONFIG_FILE` -> `/var/lib/netdog/net.toml`
* `DEFAULT_NET_CONFIG_FILE` -> `/var/lib/bottlerocket/net.toml`

Reply:
+1 to that - great idea.